File: Data Structures & Algorithms/binary-search/submission-1.py
class Solution:
    def search(self, nums: List[int], target: int) -> int:
        N = len(nums)
        for i in range(N):
            if nums[i] < target:
                continue
            elif nums[i] > target:
                return -1
            else:
                return i
        return -1

File: Data Structures & Algorithms/kth-largest-element-in-an-array/submission-1.cpp
class Solution {
public:
    int findKthLargest(vector<int>& nums, int k) {
        std::priority_queue<int, std::vector<int>, std::greater<int>> min_pq;
        for (int i : nums){
            min_pq.push(i);
            if (min_pq.size() > k){
                min_pq.pop();
            }
        }
        return min_pq.top();
    }
};
